### PR TITLE
Adds search form to #show view

### LINKED ISSUE #150: digital clips collection page: add search form in sidebar

### PR DESCRIPTION
**Pull request by afred**
Parameterizes the placeholder for the search partial.
Isolates search form style.
Closes #150.

### CODE REVIEW

**Comment by afred** [comment]
@mccalluc please review

**Comment by afred** [comment]
looks like this:
image

File: app/assets/stylesheets/locals/search_form.css.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+input {
+  background-color: whitesmoke;
```
Comment (mccalluc) [inline app/assets/stylesheets/locals/search_form.css.scss]: Reason to change background color? Delete variable if we won't use it.

**Comment by mccalluc** [comment]
(And actually, #150 is about the collection page, not the catalog details page. These got conflated during the discussion, but we need this, too. I'll make a PR for that.)

**Comment by mccalluc** [comment]
@afred: review?

**Pull request merged**